doc/release_notes.qbk

```diff
@@ -23,9 +23,9 @@
 [heading Boost 1.81]
 [/=================]
 
-[*Major improvements]
+[*Solved issues]
 
-* [@https://github.com/boostorg/geometry/pull/1045 1045] Support geographic buffer for linestrings (side, joins)
+* [@https://github.com/boostorg/geometry/issues/1076 1076] Union: in rare cases it might miss one polygon
```

Comment:
@vissarion if you cherry-pick this commit (together with the other 2), then it should be fine.